source/MaterialXCore/Definition.cpp

```diff
@@ -186,13 +186,28 @@ StringVec TargetDef::getMatchingTargets() const
 vector<UnitDefPtr> UnitTypeDef::getUnitDefs() const
```

Comment:
With the latest `dataLibrary` argument to `getChildrenOfType`, can this update to `getUnitDefs` now be simplified to use it as well?

I'm imagining something along the lines of the following:

```
vector<UnitDefPtr> UnitTypeDef::getUnitDefs() const
{
    vector<UnitDefPtr> unitDefs;

    for (UnitDefPtr unitDef : getDocument()->getChildrenOfType<UnitDef>(EMPTY_STRING, getDocument()->getDataLibrary()))
    {
        if (unitDef->getUnitType() == _name)
        {
            unitDefs.push_back(unitDef);
        }
    }

    return unitDefs;
}
```

Essentially we would now just be adding the new `dataLibrary` argument to the existing call, and making no other changes to the method.